trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/credentials/http/TestHttpCredentialsProvider.java

```diff
@@ -186,74 +180,12 @@ private void testNoCredentialsRetrieved(String emulatedAccessKey, Optional<Strin
         assertThat(httpCredentialsServlet.getRequestCount()).isEqualTo(1);
     }
 
-    private static TestingHttpServer createTestingHttpCredentialsServer(HttpCredentialsServlet servlet)
+    protected static TestingHttpServer createTestingHttpCredentialsServer(Servlet servlet)
```

Comment:
this could be public and moved to `TestingUtils` or some other class since its not specific to the credentials server. we have another function called `createTestingHttpErrorResponseServer` which does the same thing and could be replaced by this